package.json

```diff
@@ -40,12 +40,12 @@
     "@babel/register": "^7.9.0",
     "axios-mock-adapter": "^1.18.1",
     "dotenv": "^8.2.0",
-    "eslint": "^8.2.0",
+    "eslint": "^6.8.0",
```

Comment:
Why the downgrade? If the older version should be kept, I suggest to pin it, so no-one else updates it again later.